support-frontend/assets/pages/weekly-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -0,0 +1,223 @@
+// @flow
+
+// ----- Imports ----- //
```

Comment:
I think it might be worth calling this `weeklyCheckoutForm.jsx` as we currently have 3 other jsx files and one js file called checkout! We could then rename the equivalent files in paper and digital, but that could be a separate PR